tools/code_analyzer/op_deps_pass.cpp

```diff
@@ -770,6 +770,12 @@ class OpDependency : public ModulePass {
 
   static void extractStringValue(
       Value* V, const std::function<void(const std::string&)>& CB) {
+    if (isa<UndefValue>(V)) {
+      // UndefValue inherits from ConstantValue, but don't contain any data
+      // See: https://llvm.org/docs/LangRef.html#undefined-values
+      return;
+    }
```

Comment:
It turns out clang was generating an IR node type that it wasn't before, and it was causing infinite recursion.